sass/directives/_visual-guides.scss

```diff
@@ -0,0 +1,128 @@
+/* A simple, center dividing column guide / line. See /recruiting-solutions/small-business-subscription-plans-and-pricing
+ * The guide can extend out of its relatively positioned parent container on the bottom and top
+ * This guide does not include steps. It is only a center line.
+ * You must have a parent container (relatively positioned) for this to work properly.
+ */
+@mixin visual-guide-divider($extend-top: 0px, $extend-bottom: 0px) {
+
+  @if $extend-top != 0px {
+    bottom: -($extend-bottom);
+    top: -($extend-top);
+  } @else {
+    bottom: $extend-bottom;
+    top: $extend-top;
+  }
+
+  left: 0;
```

Comment:
It does work on my end. You are correct in that the values (mainly the width of the wrapper) does not matter in this case as the internal center divider has self-centered margin already applied.